entrypoint.sh

```diff
@@ -20,6 +20,7 @@ if [ ! -e "/opt/server/version" ]; then
   echo "Version $SIT_VERSION installed."
   # will prevent setup from running again
   echo "$SIT_VERSION" > /opt/server/version
+  exit 0
```

Comment:
Maybe we just drop the `exit 0` entirely here, actually? We probably still want the entrypoint to run the `CMD` (typically `Aki.Server.exe`) after the one-time setup is complete, rather than just exiting.

Reply:
NACK
config files have to be generated first, most common flow is to do first start up then let users add mods / change settings and boot again.

Reply:
I disagree pretty strongly here. Folks can download and run SIT manually if they want, but I think we should treat the docker image as an "advanced" use-case for running a production service in a managed environment (VPS or otherwise). Seeing the container start up successfully is confirmation that there weren't any unexpected / unreported issues during setup.

Needing to run the container, have it shut itself down, and then have to run the container a second time for it to actually run the `CMD` that it's been given would just 1) look like a bug (did it crash? where's my service?); and 2) would be a nuisance to automate around because normally you'd daemonize the service (through compose or otherwise) and it'd be running in the background asynchronously.

Thankfully, I think this is _mostly_ a moot point since setting `restart: unless-stopped` (or `always`) in a docker compose file will just mean it'd auto-restart immediately anyway.

Reply:
I'm indifferent on this, i can just un-suppress verbosity of the initial boot if a more clear user indication of a working deployment is required, but I think user agency is available as you mentioned regarding restart rules, those that want to daemonize can and it already is out of the box via the provided docker-compose. I'll leave it to @devbence for the tie breaker 😂